adrg/_adrg.qmd

```diff
@@ -1,9 +1,591 @@
+                │       └── programs
+                │           ├── adadas.r
+                │           ├── adae.r
+                │           ├── adlbc.r
+                │           ├── adsl.r
+                │           ├── adtte.r
+                │           ├── renv-lock.txt
+                │           ├── tlf-demographic.r
+                │           ├── tlf-efficacy.r
+                │           ├── tlf-kmplot.r
+                │           └── tlf-primary.r
+                └── tabulations
+                    └── sdtm
+                        ├── ae.json
+                        ├── cm.json
+                        ├── dm.json
+                        ├── ds.json
+                        ├── ex.json
+                        ├── lb.json
+                        ├── mh.json
+                        ├── qs.json
+                        ├── relrec.json
+                        ├── sc.json
+                        ├── se.json
+                        ├── suppae.json
+                        ├── suppdm.json
+                        ├── suppds.json
+                        ├── supplb.json
+                        ├── sv.json
+                        ├── ta.json
+                        ├── te.json
+                        ├── ti.json
+                        ├── tv.json
+                        └── vs.json
+
 
 # Appendix 1: Pilot 5 R Environment Installation and Usage {#sec-a1}
 
-More to come!
+To execute the R programs included in this Pilot, follow all of the procedures below. Ensure that you note the location of where you downloaded the Pilot 5 eCTD submission files. For demonstration purposes, the procedures below assume the transfer has been saved to this location: `C:\pilot5`.
+
+In addition, create a new directory to hold the unpacked Pilot 5 data files and associated programs. For demonstration purposes, the procedures below assume the new directory is this location: `C:\pilot5-files`.
+
+## Installation of R and RStudio
+
+Download and install R 4.4.1 for Windows from <https://cloud.r-project.org/bin/windows/base/old/4.4.1/R-4.4.1-win.exe>. While optional, it is also recommended to use RStudio IDE for executing R code to launch the application. You can download RStudio for Windows by visiting <https://posit.co/download/rstudio-desktop/#download>.
+
+## Installation of Rtools
+
+Due to certain R packages requiring compilation from source, it is also required that you install the **Rtools** Windows utility from CRAN. You can download Rtools built for R version `4.4.1` by visiting <https://cloud.r-project.org/bin/windows/Rtools/rtools44/files/rtools44-6459-6401.exe>. During the installation procedure, keep the default choices in the settings presented in the installation dialog.
+
+Once the installation is complete, launch a new R session (if you have an existing session open, close that session first) and in the console, run the following command that should give the location of your Rtools installation:
+
+``` r
```

Comment:
Hey @rpodcast wondering if this is needed?  It feels a bit confusing for a newbie.  I don't see this in the pilot3 adrg